index.html

```diff
@@ -294,9 +294,42 @@ <h1> Skils</h1>
             </div>
         </section>
         <!-- end of about -->
-        <!-- <section  class="get-in-touch">
-            <h1>Get in</h1>
-        </section> -->
+        <section  class="get-in-touch">
+            <h3 hidden>heading</h3>
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form action="/" method ="post">
+                <div class="input-control">
+                    <input class="fullname-control" type="text" id="full-name" placeholder="Full name" required>
+                </div>
+                <div class="input-control">
+                    <input class="emaol-control" type="email" id="email" placeholder="Email address" required>
+                </div>
+                <div class="input-control">
+                    <textarea name="content" id="contents"   rows="4" placeholder="Enter text here" required></textarea>
+                </div>
+                <div class="get-in-touch-button">
+                    <div hidden></div>
+                <button  class="btn-get-in-touch" type="submit">Get in touch</button>
+                    <div hidden></div>
+                </div>
+            </form>
+        </section>
+
+        <section class="footer">
```

Comment:
- [ ] Instead of using the `section` tag here, the `footer` tag is best to use as a semantic tag. Kindly use the `footer` tag here.

Reply:
This is OK, i will fix that.